components/Web_driver/Web_driver.c

```diff
@@ -381,8 +381,6 @@ static esp_err_t download_get_handler(httpd_req_t *req)
 static esp_err_t delete_post_handler(httpd_req_t *req)
 {
     char filepath[FILE_PATH_MAX];
-    FILE *fd = NULL;

```

Comment:
This will throw error if LittleFS or SPIFFS is used. Look at line 416.

Reply:
Thanks for review. Updated.